notebooks/05-visualize-distributions-boxplots.ipynb

```diff
@@ -0,0 +1,353 @@
+{

```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.        )
</pre><p>BTW I think you are seeing everything plotted on a single year location because you are passing <code>year</code> to <code>ridge</code> which is just whatever the last value of <code>year</code> was in the last iteration of the previous loop. Instead you'd want to pass <code>kde_results["year"]</code>       </p><p> </p><p>But my actual suggestion is to simplify things into a single loop. Then there is no need to maintain and store <code>kde_results</code> etc, just call <code>ridge</code> and <code>patch</code> in the same loop over the groups, and you can use <code>year</code> because it is one of the loop variables of that loop and will update every iteration.  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>